manifests/config/entry.pp

```diff
@@ -39,7 +39,7 @@
   if (!defined(Concat::Fragment["puppet.conf_${section}_${key}"])){
     concat::fragment{"puppet.conf_${section}_${key}":
```

Comment:
This fragment will always get a newline now.  This breaks adding multiple values to the same key via different instances of `puppet::config::entry` (the resources added in the `else` below).